modules/redis/src/main/scala/zio/redis/api/SortedSets.scala

```diff
@@ -580,6 +769,164 @@ trait SortedSets[G[+_]] extends RedisEnvironment[G] {
       }
     }
 
+  /**
+   * Returns the specified range of elements in the sorted set stored at <key>.
+   *
+   * @param key
+   *   Key of a sorted set
+   * @param range
+   *   ZRANGE can perform different types of range queries: by index (rank), by the score, or by lexicographical order.
+   * @param rev
+   *   The optional REV argument reverses the ordering, so elements are ordered from highest to lowest score, and
+   *   score ties are resolved by reverse lexicographical ordering.
+   * @param limit
+   *   The optional LIMIT argument can be used to obtain a sub-range from the matching elements (similar to SELECT LIMIT offset, count in SQL)
+   * @return
+   *   A list of members in the specified range with, optionally, their scores when the WITHSCORES option is given.
+   */
+  final def zRange[K: Schema](
```

Comment:
Since we're adding 7.x API, maybe we should remove the old `zRange` function and all "deprecated" functions (or at least "implement" them in terms of `zRange` and mark as `deprecated`)? WDYT?

Reply:
@mijicd Ok I deprecated the methods and implemented then in terms of the new zRange command, but CI/CD is failing because of the use of deprecated methods 🤔  How do I fix that ?

Reply:
I think the only way to fix that (and keep the deprecation notes) is to live up to our suggestions, a.k.a. update affected calls to new `zRange`. Alternatively, we could live with ignoring warnings, but that's just postponing the previous approach.

Reply:
Just change the calls in tests, don't delete.